src/applications/gi/reducers/profile.js

```diff
@@ -35,10 +36,15 @@ export default function (state = INITIAL_STATE, action) {
       };
     case FETCH_PROFILE_SUCCEEDED:
       const camelPayload = camelCaseKeysRecursive(action.payload);
-      const attributes = normalizedAttributes({
+      const bahGrandfathered = _.get(action, 'zipRatesPayload.data.attributes.mhaRateGrandfathered');
+      const attributes = {
+        ...normalizedAttributes({
         ...camelPayload.data.attributes,
         ...camelPayload.data.links
-      });
+      }),
```

Comment:
This indentation reads a little odd to me.

Reply:
fixed